src/pages/support/faq.md

```diff
@@ -6,311 +6,196 @@ import RetryDoc from '../common/retry-doc.md'
+### Why do I see duplicate fields for recipient client id and event id in the delivered payload?
 
-1. Select Project overview in the left-side links menu.
-2. Click on the "Download" button from the top menu on the right to download the project metadata json file. Open the downloaded json file with your favorite editor, and make note of the following:
-   1. Your consumer organization id (at `project.org.id`)
-   2. Your project id (at `project.id`)
-   3. Your workspace id (at `project.workspace.id`)
-3. Using the [provider API](../guides/api/provider-api.md), fetch [your I/O Events providers entitled to the provided organization id](../api.md#operation/getProvidersByConsumerOrgId), using the consumer organization id noted from above.
-4. Find the conflicting provider against your workspace id (found at `project.workspace.id` from the project json file) from the provider API response, and make a note of the provider `id`.
-5. Delete the provider via the [provider API](../api.md#operation/deleteProvider), using the ids noted in above steps.
-6. Repeat the above steps for all conflicting event providers and try deleting the project again. Your project deletion should now go through successfully.
+Adobe I/O Events sends cloud event payloads with custom attributes:
 
-### Why do I see duplicate fields in the delivered payload for attributes recipient client id and event id?
+- `event_id` (for event tracking and debugging)
+- `recipient_client_id` (for payload verification)
 
-I/O Events sends cloud event payloads with internal custom attributes like `event_id` (used for event tracking and debugging) and `recipient_client_id` (used by consumers for payload verification, see [securtiy verification guide](../guides/index.md#improved-and-resilient-security-verification-for-webhook-events)).
+After upgrading to Java 17 and the latest CloudEvents SDK, stricter attribute naming rules are enforced.
+The old names (`event_id`, `recipient_client_id`) do not conform to the [CloudEvents spec](https://github.com/cloudevents/spec/blob/v1.0/spec.md#attribute-naming-convention).
 
-As part of our recent upgrade to `Java 17`, we have adopted the latest version of the CloudEvents SDK. This version enforces stricter validation rules for custom attribute names, as outlined in the [CloudEvents spec](https://github.com/cloudevents/spec/blob/v1.0/spec.md#attribute-naming-convention). The previous attribute names (`event_id` and `recipient_client_id`) do not conform to these rules and cause serialization failures.
+**To ensure compatibility:**
 
-To ensure backward compatibility while rolling out our Java 17 upgrade, we are now including two additional attributes in the payload namely `eventid` and `recipientclientid` conforming to the cloud events spec.
+- New attributes `eventid` and `recipientclientid` are now included (CloudEvents-compliant).
```

Comment:
Should we move these also on top? Like put them together in the list with the old ones?